loopout.go

```diff
@@ -52,11 +52,32 @@ const (
 	DefaultHtlcConfTarget = 6
 
 	// DefaultSweepConfTargetDelta is the delta of blocks from a Loop Out
-	// swap's expiration height at which we begin to use the default sweep
-	// confirmation target.
-	//
-	// TODO(wilmer): tune?
-	DefaultSweepConfTargetDelta = DefaultSweepConfTarget * 2
+	// swap's expiration height at which we begin to cap sweep confirmation
+	// target with urgentSweepConfTarget and multiply feerate by factor
+	// urgentSweepConfTargetFactor.
+	DefaultSweepConfTargetDelta = 10
+
+	// urgentSweepConfTarget is the confirmation target we'll use when the
+	// loop-out swap is about to expire (<= DefaultSweepConfTargetDelta
+	// blocks to expire).
+	urgentSweepConfTarget = 3
+
+	// urgentSweepConfTargetFactor is the factor we apply to fee-rate of
+	// loop-out sweep if it is about to expire.
+	urgentSweepConfTargetFactor = 1.1
+
+	// highPrioSwapAmount is a threshold at which a swap is prioritized to
+	// be swept with a fee that is related to its size. The fee amount is
+	// determined by highPrioFeePPM. Current values correspond to 1 BTC
+	// and 0.05% proportional fee.
+	highPrioSwapAmount = 100_000_000
+
+	// highPrioFeePPM expresses the fraction of the swap amount that is paid
+	// towards sweeping fees. This fee is only supposed to be paid for swaps
+	// greater or equal the size of highPrioSwapAmount to expedite the
+	// availability of on-chain funds to generate revenue on loop-out
+	// operations.
+	highPrioFeePPM = 50
```

Comment:
While this default is sane, I'd still recommend it (as well as high prio swap amount) to be configurable.

Reply:
+1

I'll do it. Also need to make sure high prio fee is not lower than a regular fee (which is more likely to happen given the "high prio" parameters will be configurable).

Reply:
I added a flag in SweepFeeProvider which disabled high prio logic altogether. It is enabled in Loop client. So we don't need options to configure these parameters.